src/components/MultipleAvatars.js

```diff
@@ -44,50 +49,65 @@ const MultipleAvatars = (props) => {
     }
 
     if (props.avatarImageURLs.length === 1) {
+        const tooltipToPass = _.isArray(props.tooltip) ? props.tooltip.join(', ') : props.tooltip;
         return (
             <View style={avatarContainerStyles}>
-                <Avatar
-                    source={props.avatarImageURLs[0]}
-                    size={props.size}
-                    isChatRoom={props.isChatRoom}
-                    isArchivedRoom={props.isArchivedRoom}
-                />
+                <Tooltip text={tooltipToPass}>
+                    <Avatar
+                        source={props.avatarImageURLs[0]}
+                        size={props.size}
+                        isChatRoom={props.isChatRoom}
+                        isArchivedRoom={props.isArchivedRoom}
+                    />
+                </Tooltip>
             </View>
         );
     }
 
+    const tooltip = props.tooltip;
+    const tooltipForFirstAvatar = _.isArray(tooltip) ? tooltip[0] : tooltip;
+    const tooltipForAvatarCount = _.isArray(tooltip) && tooltip.length > 1 ? tooltip.slice(1).join(', ') : '';
+
     return (
-        <View pointerEvents="none" style={avatarContainerStyles}>
```

Comment:
this change? It does not seem required but let's do some testing on this and see why it was added [here](https://github.com/Expensify/App/pull/3129/files)

Reply:
If I don't remove `pointerEvents="none"`, the Tooltip doesn't show up.

Reply:
No with this prop being removed though, can you replicate this with 3D touch: https://github.com/Expensify/App/issues/3019? 

I mean 3D touch is no longer supported on the newest iphones so if this is the main concern, _maybe_ we could not worry about it. But better to not introduce new regressions.

Reply:
Yup. I agree with @mountiny. @mananjadhav Could you please do some testing for #3019. If it is causing issues then we can talk about alternatives.

Reply:
Thanks for pointing out the related GH issue. I'll test it out.

Reply:
Any update here @mananjadhav

Reply:
Did hide the tooltip for the workspace section. Missed this one to test. I'll check by tonight and update.

Reply:
I tested this and nothing broke. But I am not too sure about reproducing the issue. I have checked this on iPhone 8, which says has 3D Touch-enabled, but I couldn't find this setting.

Is there any other way to test it?

<img width="331" alt="Screenshot 2022-02-20 at 11 21 55 PM" src="https://user-images.githubusercontent.com/3069065/154856642-bbef46dd-e7c8-4cdf-8b92-549276dc97df.png">
<img width="337" alt="Screenshot 2022-02-20 at 11 13 20 PM" src="https://user-images.githubusercontent.com/3069065/154856649-7cf0f6a6-1253-4dec-a01a-2d9df9be61a9.png">

Reply:
I guess in general Long press on newer iPhones too since 3D touch has been deprecated and they use Long press instead. I believe it might have similar behaviour.

Reply:
I tried Long press on iPhone 12 and iPhone 8, for which I've uploaded the screenshots above.

Reply:
However, I think the long press will be fine and will do what you have shown on the screens in the thread. So I guess this is fine.

Reply:
Thanks, @mananjadhav for looking into this. Let's add QA steps to test https://github.com/Expensify/App/issues/3019. Do mention to test https://github.com/Expensify/App/issues/3019.